plugins/chat/config/locales/client.en.yml

```diff
@@ -289,8 +289,10 @@ en:
         always: "For all activity"
 
       settings:
-        enable_auto_join_users: "Automatically add all recently active users"
-        disable_auto_join_users: "Stop automatically adding users"
+        auto_join_users_label: "Automatically add users"
+        auto_join_users_info: "Check hourly which users have been active in the last 3 months and, if they have access to the %{category} category, add them to this channel."
```

Comment:
The only possible worry I have with this is that, if we change the logic in `Jobs::AutoManageChannelMemberships` or `Jobs::AutoJoinUsers` we need to remember to update this. It's overkill to have to pass these values into the translation, so maybe just add a note to the top of both the `plugins/chat/app/jobs/regular/auto_manage_channel_memberships.rb` ruby class, e.g. something like:

```
NOTE: When changing auto-join logic, make sure to update the `settings.auto_join_users_info` translation as well.